PokepayTests/PokepayTests.swift

```diff
@@ -572,6 +575,9 @@ AQIDAQAB
                         if response.transaction != nil {
                             XCTFail("transaction should be null")
                         }
+                        if response.metadata == nil {
+                            XCTFail("metadata should not be null")
+                        }
```

Comment:
@WeiWW can you add a confirmation that response.metadata matches the requested object?

Reply:
modified it.